Dockerfile

```diff
@@ -0,0 +1,25 @@
+ARG PYTHON_VERSION=3.12
+FROM python:${PYTHON_VERSION}-slim AS base
+LABEL maintainer="vlados.soc@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+RUN mkdir -p /files/media
+
+RUN adduser \
+    --disabled-password \
+    --no-create-home \
+    my_user

```

Comment:
It's a good practice to specify a UID and GID when creating a user to avoid potential conflicts with existing users in the base image. Consider adding `--uid` and `--gid` options to the adduser command.